bin/runner.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
@kilemensi  Instead of having all these under one Dockerfile, I have a second implementation where each of the scripts run on their own docker containers thus we can have 3 different Dockerfiles. Which approach do you think is better?

Reply:
Not sure what these are @esirK . If they're completely different applications and this is some sort of micro-services architecture then they, different Dockerfiles would make some sense.

But I have a feeling these are just processes of the same app and hence a [Procfile](https://dokku.com/docs~v0.24.0/deployment/builders/dockerfiles/#procfiles-and-multiple-processes) could be all that's needed.

Reply:
Basically, they are processes of the same app. Was thinking of making them some sort of microservices but if the Procfile will help then I'll use that.